mw/auth/jwt_test.go

```diff
@@ -0,0 +1,83 @@
+package auth
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"google.golang.org/grpc/metadata"
+)
+
+const (
+	TEST_SECRET = "some-secret-123"
+)
+
+func TestGetTenantID(t *testing.T) {
+	var tenantIDTests = []struct {
+		claims   jwt.MapClaims
+		field    string
+		expected string
+		err      error
+	}{
+		{
+			claims: jwt.MapClaims{
+				"some-field": "id-abc-123",
+			},
+			field:    "some-field",
+			expected: "id-abc-123",
+			err:      nil,
+		},
+		{
+			claims: jwt.MapClaims{
+				"some-field": "id-abc-123",
+			},
+			field:    "some-other-field",
+			expected: "",
+			err:      errMissingField,
+		},
+		{
+			claims:   jwt.MapClaims{},
+			field:    "some-field",
+			expected: "",
+			err:      errMissingToken,
+		},
+	}
+	for _, test := range tenantIDTests {
```

Comment:
this doesn't seem to test the `GetAccountID` function anymore at all, instead testing the `GetJWTField` function.

You should also have tests for the `GetAccountID` function -- it can be a small, easy test.

Reply:
Makes sense!